packages/terra-time-input/src/TimePipe.js

```diff
@@ -0,0 +1,26 @@
+export default function TimePipe(timeFormat = 'HH:mm') {
```

Comment:
I think we want to remove the default format. It will not always be the same default based on the locale.